Purchases/Public/RCPurchasesErrors.h

```diff
@@ -32,26 +32,27 @@ extern NSErrorUserInfoKey const RCReadableErrorCodeKey NS_SWIFT_NAME(Purchases.R
  */
 typedef NS_ERROR_ENUM(RCPurchasesErrorDomain, RCPurchasesErrorCode) {
     RCUnknownError = 0,
-    RCPurchaseCancelledError,
-    RCStoreProblemError,
-    RCPurchaseNotAllowedError,
-    RCPurchaseInvalidError,
-    RCProductNotAvailableForPurchaseError,
-    RCProductAlreadyPurchasedError,
-    RCReceiptAlreadyInUseError,
-    RCInvalidReceiptError,
-    RCMissingReceiptFileError,
-    RCNetworkError,
-    RCInvalidCredentialsError,
-    RCUnexpectedBackendResponseError,
-    RCInvalidAppUserIdError,
-    RCOperationAlreadyInProgressError,
-    RCUnknownBackendError,
-    RCInvalidAppleSubscriptionKeyError,
-    RCIneligibleError,
-    RCInsufficientPermissionsError,
-    RCPaymentPendingError,
-    RCInvalidSubscriberAttributesError,
+    RCPurchaseCancelledError = 1,
+    RCStoreProblemError = 2,
+    RCPurchaseNotAllowedError = 3,
+    RCPurchaseInvalidError = 4,
+    RCProductNotAvailableForPurchaseError = 5,
+    RCProductAlreadyPurchasedError = 6,
+    RCReceiptAlreadyInUseError = 7,
+    RCInvalidReceiptError = 8,
+    RCMissingReceiptFileError = 9,
+    RCNetworkError = 10,
+    RCInvalidCredentialsError = 11,
+    RCUnexpectedBackendResponseError = 12,
+    RCReceiptInUseByOtherSubscriberError __attribute((deprecated("Use RCReceiptAlreadyInUseError."))) = 13,
+    RCInvalidAppUserIdError = 14,
+    RCOperationAlreadyInProgressError = 15,
+    RCUnknownBackendError = 16,
+    RCInvalidAppleSubscriptionKeyError = 17,
+    RCIneligibleError = 18,
+    RCInsufficientPermissionsError = 19,
+    RCPaymentPendingError = 20,
+    RCInvalidSubscriberAttributesError = 21,
```

Comment:
@vegaro numbers added. they're in the same order that they were before the PR, with the only addition being the error that we had previously removed. I added it to the place it was before we did the logging changes.

Reply:
Thanks so much for doing that